From c0473859a4405d48e3e3ac310a735ca9c50c9e0c Mon Sep 17 00:00:00 2001
From: Prabhakar Kushwaha <prabhakar.kushwaha@nxp.com>
Date: Thu, 7 Apr 2016 09:26:12 +0800
Subject: [PATCH 025/245] arm64/config: IFC config enablement

commit d7e4a0bc1825dd212df2e30741f7be293102cf99
[context adjustment]

Adding the required configs for IFC for SDK 2.0

Signed-off-by: Raghav Dogra <raghav.dogra@nxp.com>
Signed-off-by: Prabhakar Kushwaha <prabhakar.kushwaha@nxp.com>
Integrated-by: Zhao Qiang <qiang.zhao@nxp.com>
---
 arch/arm64/configs/freescale.config | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/arch/arm64/configs/freescale.config b/arch/arm64/configs/freescale.config
index 9f149ea..6e667c8 100644
--- a/arch/arm64/configs/freescale.config
+++ b/arch/arm64/configs/freescale.config
@@ -21,3 +21,18 @@ CONFIG_CLK_QORIQ=y
 CONFIG_USB_XHCI_HCD=y
 CONFIG_USB_DWC3=y
 CONFIG_DMADEVICES=y
+#ifc
+CONFIG_MTD_OF_PARTS=y
+CONFIG_MTD_GEN_PROBE=y
+CONFIG_FTL=y
+CONFIG_MTD=y
+CONFIG_MTD_CMDLINE_PARTS=y
+CONFIG_MTD_BLOCK=y
+CONFIG_MTD_CFI=y
+CONFIG_MTD_CFI_ADV_OPTIONS=y
+CONFIG_MTD_CFI_INTELEXT=y
+CONFIG_MTD_CFI_AMDSTD=y
+CONFIG_MTD_CFI_STAA=y
+CONFIG_MTD_PHYSMAP_OF=y
+CONFIG_MTD_NAND=y
+CONFIG_MTD_NAND_FSL_IFC=y
-- 
2.1.0.27.g96db324

